db/data/seedDatabase.js

```diff
@@ -24,7 +25,7 @@ MongoClient.connect(url, (err, client) => {
 
     for (let i = 0; i < numBatches; i += 1) {
       fakeRestaurantsBatch = [];
-      generateFakeRestaurantsBatch(batchSizeSetting);
+      generateFakeRestaurantsBatch(batchSize);
```

Comment:
then here I would say -- you don't even need to pass this in as a parameter anymore, right? because `generateFakeRestaurantsBatch` is now grabbing that from the global scope, not from the passed-in arguments?